File: projects/project10/Ruby/Kiwi.rb
# Kiwi.rb | Defines the Kiwi class which inherits attributes and methods
#   from the WalkingBird superclass.
#
# Created by: Ethan Clark Project10
# Date: April 26, 2016
####################################################

require './WalkingBird.rb'

class Kiwi < WalkingBird

	######################################################
	# call returns the sound that a Kiwi type makes      #
	# Recieve: NONE                                      #
	# Return: Victory!, a String                         #
	######################################################
	def call
		"Victory!"
	end

end

File: projects/project10/Java/Kiwi.java
/* Kiwi.java models a Kiwi
 *
 * Created by: Ethan Clark Project10
 * Date: April 25, 2016
 */

public class Kiwi extends WalkingBird
{

	/**********************************************
	* Constructor for the child Kiwi class        *
	* Sets name to be myName in the parent class  *
	**********************************************/
	public Kiwi(String name)
	{
		super(name);
	}

	/**********************************************
	* call() returns the sound for the Kiwi class *
	* Receive: NONE                               *
	* Return: "Victory!"                          *
	**********************************************/
	public String call()
	{
		return "Victory!";
	}
}
